de.json

```
@@ -777,14 +777,14 @@
         "Settings.Metric" : "Metrisch (cm)",
         "Settings.Imperial" : "Imperial (ft)",
 
-        "Settings.OverrideLocale" : "Übersteuere Lokalisierung:",
-        "Settings.HideAllTutorials" : "Blende alle Tutorials aus",
+        "Settings.OverrideLocale" : "Überschreibe Gebietsschema:",
```

Not descriptive that we mean the language here. This sounds better
```suggestion
        "Settings.OverrideLocale" : "Überschreibe Sprachumgebung:",
```